Library/Homebrew/completions.rb

```diff
@@ -12,6 +12,14 @@ module Homebrew
   module Completions
     extend T::Sig
 
+    Variables = Struct.new(
```

Comment:
Stupid question perhaps: what makes a `Struct` ok but an `OpenStruct` not?

Reply:
An `OpenStruct` accepts any method, returning `nil` by default (i think of it as a Hash that converts dot access to bracket access, not sure how accurate that is under the hood). A `Struct` only defines methods for the args in its constructor:

https://sorbet.run/#%23%20typed%3A%20true%0A%0AFoo%20%3D%20Struct.new%28%3Afoo%29%0A%0Abar%20%3D%20Foo.new%28'a'%29%0A%0Abar.foo%0Abar.baz%0A%0Aos%20%3D%20OpenStruct.new%0Aos.foo%20%23%20valid%2C%20returns%20nil%0A

Reply:
Ok, gotcha, thanks!